article-store/articles/api.py

```diff
@@ -68,14 +75,15 @@ def post(self, request: Request, article_id: str) -> HttpResponse:
         :param article_id: str
         :return: class: `HttpResponse`
         """
-
         if not article_id or not Article.id_is_valid(article_id):
             return Response({'error': 'Please provide a valid article id'}, status=status.HTTP_406_NOT_ACCEPTABLE)
```

Comment:
`406 Not Acceptable` relates to `Accept` headers. Invalid input should be `400 Bad Request`